packages/plugin/src/Fields/FileUploadField.php

```diff
@@ -129,7 +138,16 @@ public function getInputHtml(): string
     public function uploadFile()
     {
         if (!isset(self::$filesUploaded[$this->handle])) {
-            $response = $this->getForm()->getFileUploadHandler()->uploadFile($this, $this->getForm());
+            $response = null;
+
+            if ($this->getForm()->isGraphQLPosted()) {
+                try {
+                    $response = $this->getForm()->getFileUploadHandler()->uploadGraphQL($this, $this->getForm());
+                } catch (GuzzleException|InvalidSubpathException|Exception $e) {
```

Comment:
```suggestion
                } catch (Exception $e) {
```
`Exception` covers all of the cases.